lib/migration/helpers.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Migration
+  module Helpers
+    def self.site_created_at
+      result = DB.query_single <<~SQL
+        SELECT created_at
+        FROM schema_migration_details
+        ORDER BY created_at
+        LIMIT 1
+      SQL
+      result.first
+    end
+
+    def self.old_site?
```

Comment:
From one of the comments in migrations:

> make setting enabled for existing sites

maybe "existing_site?" would be more fitting? One hour is not "old" 😛

Reply:
Good idea - updated 👍